src/hotspot/share/gc/shenandoah/shenandoahControlThread.cpp

```diff
@@ -472,12 +471,17 @@ bool ShenandoahControlThread::is_explicit_gc(GCCause::Cause cause) const {
          GCCause::is_serviceability_requested_gc(cause);
 }
 
+bool ShenandoahControlThread::is_async_gc(GCCause::Cause cause) const {
+  return cause == GCCause::_wb_breakpoint;
+}
```

Comment:
Do we really need this method? What is "async gc" anyway? I think you can just inline the method at its only use.

Reply:
Done